src/map/mob.c

```diff
@@ -3733,74 +4010,126 @@ bool mob_parse_dbrow(char** str) {
 #endif
 
 	//Tests showed that chase range is effectively 2 cells larger than expected [Playtester]
-	if (db->range3 > 0)
-		db->range3 += 2;
+	if (entry->range3 > 0)
```

Comment:
It's funny how this check (not the fault of this commit, it was already like this) is done before actually reading the value...